### DISCUSSION TITLE
Is cache folder in .next needed for production?

### DISCUSSION THREAD

**Discussion by leongkui** [vercel/next.js#32200] [Help]
In attempt to reduce the container size, I found out that the cache folder in .next are big in size.  

```
$ du -sh cache server static
1.2G	cache
1.4M	server
 22M	static
```

I tested removing `cache` folder and running using `next start`, the application seems to work.  Tested removing the cache folder in the container image also seems ok.   Question is that in running the next.js application within container environment in production,  is the cache folder used during application runtime in production mode in any way?  Is it safe to remove it?    Thanks in advance for any advice on this.

**Comment by Tokenyet** [comment]
I want to know this topic, either. Here are my tests:
1. On development, **.cache** size: 2.9GB
2. On ci/cd, new project with `next build`, **.cache** size: 136MB

According to 2, I think cache is a must, because `next build` still generate It on my pipeline. 😕

**Reply by leongkui** [reply]
thanks for sharing.  I did a test by removing the cache folder and the application works just fine.  For now, I removed the post build, this significantly reduce my container image size.  I would to have a definitive answer to this.

**Reply by pcjmfranken** [reply]
Have you checked the Vercel or Netlify build process or logs to see what they do and end up with?

**Reply by cemcakirlar** [reply]
There is a comment marked as answer here, no you do not need the folder in production. In case someone looks for an answer in this thread.
https://github.com/vercel/next.js/discussions/31395#discussioncomment-1643499

**Reply by rpratama-codes** [reply]
@cemcakirlar there is no XD
